src/test/scala/io/iohk/ethereum/jsonrpc/JsonRpcControllerSpec.scala

```diff
@@ -1,5 +1,9 @@
 package io.iohk.ethereum.jsonrpc
 
+import akka.actor.ActorSystem
+import akka.testkit.TestProbe
+import io.circe.Json.JString
```

Comment:
We're using json4s, aren't we? I believe circe shouldn't even be in the dependencies anymore (sadly!)